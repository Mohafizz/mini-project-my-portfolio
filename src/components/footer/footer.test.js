import React from "react";
import { shallow } from "enzyme";
import MyFooter from "./footer";

const footer = shallow(<MyFooter />);

it("should render correctly", () => {
  expect(footer).toMatchSnapshot();
});
