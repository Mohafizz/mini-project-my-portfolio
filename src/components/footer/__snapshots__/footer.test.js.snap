// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <MyFooter />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <Footer
        size="mini"
>
        <LinkList
                size="mega"
        >
                <IconButton
                        href="/"
                        name="home"
                />
                <IconButton
                        href="https://www.linkedin.com/in/mohamed-hafiz-796251142/"
                        name="accessibility"
                        target="blank"
                />
                <IconButton
                        href="mailto:mohafizz@gmail.com"
                        name="mail"
                        target="blank"
                />
                Travis CI:
                <img
                        alt="travis-ci"
                        src="https://travis-ci.org/Mohafizz/mini-project-my-portfolio.svg?branch=master"
                />
        </LinkList>
</Footer>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": <LinkList
          size="mega"
>
          <IconButton
                    href="/"
                    name="home"
          />
          <IconButton
                    href="https://www.linkedin.com/in/mohamed-hafiz-796251142/"
                    name="accessibility"
                    target="blank"
          />
          <IconButton
                    href="mailto:mohafizz@gmail.com"
                    name="mail"
                    target="blank"
          />
          Travis CI:
          <img
                    alt="travis-ci"
                    src="https://travis-ci.org/Mohafizz/mini-project-my-portfolio.svg?branch=master"
          />
</LinkList>,
        "size": "mini",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "children": Array [
            <IconButton
              href="/"
              name="home"
/>,
            <IconButton
              href="https://www.linkedin.com/in/mohamed-hafiz-796251142/"
              name="accessibility"
              target="blank"
/>,
            <IconButton
              href="mailto:mohafizz@gmail.com"
              name="mail"
              target="blank"
/>,
            "Travis CI:",
            <img
              alt="travis-ci"
              src="https://travis-ci.org/Mohafizz/mini-project-my-portfolio.svg?branch=master"
/>,
          ],
          "size": "mega",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "href": "/",
              "name": "home",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "href": "https://www.linkedin.com/in/mohamed-hafiz-796251142/",
              "name": "accessibility",
              "target": "blank",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "href": "mailto:mohafizz@gmail.com",
              "name": "mail",
              "target": "blank",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          "Travis CI:",
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "alt": "travis-ci",
              "src": "https://travis-ci.org/Mohafizz/mini-project-my-portfolio.svg?branch=master",
            },
            "ref": null,
            "rendered": null,
            "type": "img",
          },
        ],
        "type": [Function],
      },
      "type": [Function],
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <Footer
          size="mini"
>
          <LinkList
                    size="mega"
          >
                    <IconButton
                              href="/"
                              name="home"
                    />
                    <IconButton
                              href="https://www.linkedin.com/in/mohamed-hafiz-796251142/"
                              name="accessibility"
                              target="blank"
                    />
                    <IconButton
                              href="mailto:mohafizz@gmail.com"
                              name="mail"
                              target="blank"
                    />
                    Travis CI:
                    <img
                              alt="travis-ci"
                              src="https://travis-ci.org/Mohafizz/mini-project-my-portfolio.svg?branch=master"
                    />
          </LinkList>
</Footer>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "children": <LinkList
            size="mega"
>
            <IconButton
                        href="/"
                        name="home"
            />
            <IconButton
                        href="https://www.linkedin.com/in/mohamed-hafiz-796251142/"
                        name="accessibility"
                        target="blank"
            />
            <IconButton
                        href="mailto:mohafizz@gmail.com"
                        name="mail"
                        target="blank"
            />
            Travis CI:
            <img
                        alt="travis-ci"
                        src="https://travis-ci.org/Mohafizz/mini-project-my-portfolio.svg?branch=master"
            />
</LinkList>,
          "size": "mini",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": Array [
              <IconButton
                href="/"
                name="home"
/>,
              <IconButton
                href="https://www.linkedin.com/in/mohamed-hafiz-796251142/"
                name="accessibility"
                target="blank"
/>,
              <IconButton
                href="mailto:mohafizz@gmail.com"
                name="mail"
                target="blank"
/>,
              "Travis CI:",
              <img
                alt="travis-ci"
                src="https://travis-ci.org/Mohafizz/mini-project-my-portfolio.svg?branch=master"
/>,
            ],
            "size": "mega",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "href": "/",
                "name": "home",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "href": "https://www.linkedin.com/in/mohamed-hafiz-796251142/",
                "name": "accessibility",
                "target": "blank",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "href": "mailto:mohafizz@gmail.com",
                "name": "mail",
                "target": "blank",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            "Travis CI:",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "alt": "travis-ci",
                "src": "https://travis-ci.org/Mohafizz/mini-project-my-portfolio.svg?branch=master",
              },
              "ref": null,
              "rendered": null,
              "type": "img",
            },
          ],
          "type": [Function],
        },
        "type": [Function],
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
